File: src/com/designpatterns/builder/BuilderTest.java
package com.designpatterns.builder;
import java.io.IOException;

public class BuilderTest {
    public static void main(String args[]) throws IOException {
        Person om =new  PersonBuilder()
                .withName("Omkar")
                .withAge(25)
                .address()
                    .withCity("Lucknow")
                    .withState("Uttar Pradesh")
                .work()
                    .withCompany("Amazon")
                    .withDesignation("Frontend Engineer")
                .build();
        System.out.println(om);
    }
}

File: js/Builder/BuilderTest.js
const {PersonBuilder}=require('./PersonBuilder');

const Ram=new PersonBuilder()
                            .withName('Omkar')
                            .withAge('24')
                            .address()
                                .withState('UP')
                                .withCity('Lucknow')
                             .work()
                                .withCompany('Random')
                                .build();

console.log(Ram.toString());